### DISCUSSION TITLE
How to set unique meta descriptions for each page?

### DISCUSSION THREAD

**Discussion by ghost** [nunocoracao/blowfish#463] [Q&A] [answered]
I've been racking my brain about this, and I think it has something to do with the partials files but cannot figure it out. 

Does anyone know how to set unique meta descriptions in Blowfish?

**Comment by nunocoracao** [comment, marked as answer]
Use the `summary` frontmatter field. Take a look at this example from my personal website:

- link: https://nunocoracao.com/posts/202301-how-to-structure-product-teams/
- md file: https://github.com/nunocoracao/nunocoracao.com/blob/main/content/posts/202301-how-to-structure-product-teams/index.md
- Embed image attached

![Screenshot 2023-01-17 at 22 42 53](https://user-images.githubusercontent.com/3803196/213028363-2e05315d-835c-4646-9561-bd921b0bab1c.png)

**Reply by ghost** [reply]
Hey.

Thanks. But I think that is setting the og:Description, not the meta
description.

See my screenshot
![77076818-B4AF-4BEE-8344-CCD1D8BBC31B](https://user-images.githubusercontent.com/111182358/213029057-ab158f06-ad58-4590-a62d-e549a3979ebf.jpeg)

**Reply by nunocoracao** [reply]
Yes, but most websites use the one I mentioned when sharing articles. You can try the description frontmatter param. Otherwise, it's currently not supported.
